src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.h

```diff
@@ -1,17 +1,29 @@
 #ifndef Z_OBJ_TOKEI_TOBIRA_H
 #define Z_OBJ_TOKEI_TOBIRA_H
 
+#define OBJTOKEITOBIRA_GET_TYPE(thisx) ((thisx)->params & 1)
+#define OBJTOKEITOBIRA_PARAM(type) (type)
+
 #include "global.h"
 
+
 struct ObjTokeiTobira;
 
-typedef void (*ObjTokeiTobiraActionFunc)(struct ObjTokeiTobira*, PlayState*);
+typedef void (*ObjTokeiTobiraActionFunc)(struct ObjTokeiTobira*);
+
+
+typedef enum ObjTokeiTobiraType {
+    /* 0 */ OBJTOKEITOBIRA_TYPE_0,
+    /* 1 */ OBJTOKEITOBIRA_TYPE_1
+} ObjTokeiTobiraType;
 
 typedef struct ObjTokeiTobira {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x18];
+    /* 0x000 */ DynaPolyActor dyna;
     /* 0x15C */ ObjTokeiTobiraActionFunc actionFunc;
-    /* 0x160 */ char unk_160[0x10];
-} ObjTokeiTobira; // size = 0x170
+    /* 0x160 */ f32 unk160;
+    /* 0x164 */ f32 unk164;
+    /* 0x168 */ s32 unk168;
+    /* 0x16C */ s32 unk16C;
+} ObjTokeiTobira;    /* size = 0x170 */
```

Comment:
We usually use this type of comment for the size of the structs
```suggestion
} ObjTokeiTobira; // size = 0x170
```